source/Applications/Basic/FileFormats/zdfread.m

```diff
@@ -20,7 +20,7 @@
 try
     if strcmp(Filename(end-3:end),'.zdf')
         
-        disp(['Reading ',sprintf('%s',Filename),' point cloud'])
+        disp(['Reading point cloud: ',sprintf('%s',Filename)])
```

Comment:
You should not use sprintf inside disp. disp is basically sprintf('%s\n',input). So you wouldn't put a sprintf into another sprintf??
disp(['Reading point cloud: ',Filename])

Reply:
done